src/common/shibuyaHelper.js

```diff
@@ -166,6 +168,8 @@ class ShibuyaHelper {
         return location;
     };
 
+    getLocation = () => paraIdToLocation(this.config.paraId);
+
     getNativeAssetLocation = () => this.getAssetLocation(this.config.symbol);
```

Comment:
No, do not create a `symbol` for the chain config, which doesn’t really make sense. Let’s use the first element of the asset array as the native token.

so getNativeAssetLocation always returns the first symbol, and please remove the `symbol` in `this.config.symbol`

Reply:
We decided not to update this part of the code since we plan to move xcm-config to oak.js